Autopilot/PathManager/Src/CommWithAttitudeManager.cpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Implements methods for communication with the attitude manager.
+ * Author: Anthony Bertnyk
+ */
+
+#include "../Inc/CommWithAttitudeManager.hpp"
```

Comment:
Our build system takes care of resolving paths, so just #include "CommWithAttitudeManager.hpp" is fine